packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -392,6 +393,7 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
                 shouldSelectFullInputValueInComponentDidUpdate={this._onShouldSelectFullInputValueInAutofillComponentDidUpdate}
                 title={title}
                 preventValueSelection={!focused}
+                placeholder={placeholder}
```

Comment:
Can you please add tests for this new scenario?

Reply:
It’s just passing through a built in input prop to the browser. Do you still think it needs tests? (Seems like visual tests would make more sense than unit tests, to make sure it’s getting passed through correctly.)

Reply:
Yes, of course. Visual and snapshot test.

Reply:
> 
> 
> Yes, of course. Visual and snapshot test.

+1